### PR TITLE
Fixed path for loading cookiecutter.json

### PR DESCRIPTION
**Pull request by slougheed**
Fixed path when loading an existing cookiecutter.json file issue #9313. Now passing in the template path and contents path in the values JsonObject. 

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/scaffolder-backend-module-cookiecutter/src/actions/fetch/cookiecutter.ts [hunk 1]
```diff
@@ -233,6 +248,8 @@ export function createFetchCookiecutterAction(options: {
         _copy_without_render: ctx.input.copyWithoutRender,
         _extensions: ctx.input.extensions,
         imageName: ctx.input.imageName,
+        templateDir: templateDir,
+        templateContentsDir: templateContentsDir,
```
Comment (Rugvip) [inline plugins/scaffolder-backend-module-cookiecutter/src/actions/fetch/cookiecutter.ts:252]: I think think there's any reason to pass these through `values`, we might as well pass them as separate options to `run({...})` right? The values are what we end up passing to cookiecutter for templating, and while we do deconstruct and remove these from that set of values, you can still end up in confusing naming clashes.

If that's the case we should move over the `imageName` too. Will save us from all the `as string` casting as well :grin:

**Review by Rugvip — COMMENTED** [review]
Thank you! 👍 

Looks good overall, just a thought on how we're passing things around

**Comment by freben** [comment]
I'll note also, that this PR needs a changeset.

File: plugins/scaffolder-backend-module-cookiecutter/src/actions/fetch/cookiecutter.ts (hunk 1 above)
Comment (slougheed) [reply plugins/scaffolder-backend-module-cookiecutter/src/actions/fetch/cookiecutter.ts]: Agreed. Updated the code to not pass everything in through values.

**Review by Rugvip — APPROVED** [review]
Thank you, looks good! 👍 

Let's :shipit:

**Pull request merged**